backend/ml_model/use_cases/multiple_model_use.py

```diff
@@ -11,7 +13,7 @@ class EvaluateModelsUseCase:
     Use case class for evaluating multiple models for fairness.
     """
 
-    def __init__(self, model_files):
+    def __init__(self, file_repo: FileRepository, model_files):
```

Comment:
Can you elaborate on this part and explain the code?

Reply:
This is added as we need to call the function save_data_to_csv. Since we removed the data from the project, it is now stored in the SQL database and this function is used to temporarily store it in a file so that ml_model can access it. The data is then deleted again maintaining confidentially of data.